.github/workflows/release-pipeline.yaml

```diff
@@ -46,20 +46,24 @@ jobs:
       - name: Display structure of downloaded files
         run: ls -Rl ./ci-artifacts/
 
-      - name: Prepare Linux Release Asset
+      - name: Prepare Linux Release Assets
         run: |
           mkdir -p ./ci-artifacts/linux
           ./.github/scripts/releases/extract_build_linux.sh ./ci-artifacts/linux ./ci-artifacts/opengoal-linux-static ./
-          cd ci-artifacts/linux
+          pushd ci-artifacts/linux
           TAG_VAL=$(echo ${{ github.REF }} | awk -F'refs/tags/' '{print $2}')
           tar czf ../final/opengoal-linux-${TAG_VAL}.tar.gz .
+          popd
+          strip ./ci-artifacts/opengoal-linux-static/lsp/lsp
```

Comment:
True, although ive yet to figure out a decent way to debug the app.  Since the editor owns the process (starts it and such), it seems very difficult to attach a debugger to it anyway.